console-api/proto/common.proto

```diff
@@ -78,6 +78,8 @@ message Span {
     repeated Field fields = 3;
     // Timestamp for the span.
     google.protobuf.Timestamp at = 4;
+    // An Id that uniquely identifies this span's parent (if any).
+    optional SpanId parent_id = 5;
```

Comment:
in proto3, every field (that is a Message) is optional by default, so you don't need the `optional` keyword